### DISCUSSION TITLE
Globally configure API routes

### DISCUSSION THREAD

**Discussion by devth** [vercel/next.js#17821] [Help] [answered]
The [API Middlewares / Custom Config](https://nextjs.org/docs/api-routes/api-middlewares#custom-config) docs show how to configure API routes on an individual basis.

Can it be configured globally in one place too? I'd like to configure defaults explicitly for my app.

**Comment by jamesmosier** [comment, marked as answer]
Hi @devth. There is no built-in global way to do this. You'd need to import/use your middleware in each route individually.
